pkg/controllers/user/helm/common.go

```diff
@@ -58,7 +59,12 @@ func helmInstall(templateDir, kubeconfigPath string, app *v3.App) error {
 	defer cancel()
 	addr := common.GenerateRandomPort()
 	probeAddr := common.GenerateRandomPort()
-	go common.StartTiller(cont, addr, probeAddr, app.Spec.TargetNamespace, kubeconfigPath)
+	go func() {
+		err := common.StartTiller(cont, addr, probeAddr, app.Spec.TargetNamespace, kubeconfigPath)
+		if err != nil {
+			logrus.Errorf("got error [%s] while stopping tiller", err.Error())
```

Comment:
Put the error message at the end instead of inside the brackets, as it can be long.

Reply:
Updated